apps/users/models.py

```diff
@@ -145,6 +150,13 @@ def create_superuser(self, username, email, password):
         GroupUser.objects.create(user=user, group=admins)
         return user
 
+    def _generate_username(self, seed):
+        if self.model.objects.filter(username=seed).exists():
+            return '{seed}-{num}'.format(
+                seed=seed, num=random.randint(1000, 9999))
```

Comment:
Should we try doing this a few times? Also this doesn't check that `{seed}-{num}` already exists, so it will just throw a unique error if it does?